### ISSUE TITLE
Retro-Compatibility

### ISSUE THREAD

**Issue by sydneycolemanfromgoogle** [w3c/wcag3#506]
The W3C has stated that WCAG 3.0 will not be backwards compatible with the 2.x series. Organizations that spent years reaching WCAG 2.2 AA may find that their existing "compliant" code doesn't meet the new "Foundational" requirements of 3.0. This creates a risk of "Compliance Debt," where teams must re-audit and re-remediate previously "fixed" content.

**Comment by hidde** [comment]
Thanks for this feedback. 

Realistically, WCAG 2 AA status isn't forever either, and it's common in industry to re-evaluate on a regular basis either way.

Closing this issue as no open actions remain.

**Comment by alastc** [comment]
I would also add that a mapping document will be produced (once it's closer to recommendation), to make the process easier.
